src/cgroup.rs

```diff
@@ -492,6 +492,34 @@ impl Cgroup {
         v.dedup();
         v
     }
+
+    /// Checks if the cgroup exists.
+    ///
+    /// Returns true if at least one subsystem exists.
+    pub fn exists(&self) -> bool {
+        for subsystem in self.subsystems().iter() {
```

Comment:
changed it as:

let mut exist = false;

for subsystem in self.subsystems().iter() {
     exists = match subsystem {
            xxxx => c.exists()
     }  
}

exist

Reply:
Replacing with above would change the semantics of this function.

- Before: return true if one subsystem exists at least;
- After: return true if all subsystems exist.